README.md

```diff
@@ -108,9 +114,15 @@ cd kotaemon
 
 pip install -e "libs/kotaemon[all]"
 pip install -e "libs/ktem"
+
+# optional system dependencies if not installed
+brew install libmagic
```

Comment:
Could you explain why we need libmagic here? What is it used for?

Reply:
It is a system dependency of the `unstructured` package that is used to detect file types. You guys use unstructured to do all of your document parsing. The system dependencies are documented here: https://docs.unstructured.io/open-source/installation/full-installation